### PR TITLE
feat: Page not found page

### LINKED ISSUE #24: Create 404 Page

### PR DESCRIPTION
**Pull request by Miran-Firdausi**
Closes: #24 
Added page not found page
Preview:
image

### CODE REVIEW

File: src/pages/PageNotFound/index.css
Comment (Achyut-0705) [inline src/pages/PageNotFound/index.css:1]: Remove empty file

File: src/pages/PageNotFound/index.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+import error404Image from "@/assets/images/error404.svg";
+
+function PageNotFound({ text, link }) {
```
Comment (Achyut-0705) [inline src/pages/PageNotFound/index.jsx]: Define prop types

File: src/pages/PageNotFound/index.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Link } from "react-router-dom";
+import error404Image from "@/assets/images/error404.svg";
+
+function PageNotFound({ text, link }) {
+  return (
+    <div className="w-full h-[calc(100vh-4.6rem)] flex items-center justify-center text-center">
+      <div className="text-center">
+        <img
+          className="block m-auto xs:w-[90vw] sm:w-[70vw] md:w-[45vw]"
+          src={error404Image}
```
Comment (Achyut-0705) [inline src/pages/PageNotFound/index.jsx:9]: Set alt text

**Review by Achyut-0705 — CHANGES_REQUESTED** [review]
Do the necessary changes

File: src/pages/PageNotFound/index.jsx (hunk 1 above)
Comment (Achyut-0705) [inline src/pages/PageNotFound/index.jsx]: Do we need text nd link? Isn't this a static page?
If u want to add url then get it via url

**Review by Achyut-0705 — CHANGES_REQUESTED** [review]

File: src/pages/PageNotFound/index.jsx (hunk 1 above)
Comment (Miran-Firdausi) [reply src/pages/PageNotFound/index.jsx]: Accidentaly pushed the previous version, there is no need of text and link.

Comment (Miran-Firdausi) [reply src/pages/PageNotFound/index.jsx]: No props were required

**Comment by Miran-Firdausi** [comment]
I have made all the necessary changes

**Review by codexsit — APPROVED** [review]
LGTM!

**Review by Achyut-0705 — APPROVED** [review]
LGTM!

**Pull request merged**